tests/BasicTests.elm

```diff
@@ -273,6 +280,64 @@ twoChildFormDecoded =
             }
 
 
+keyedNodeWithText : Html.Html msg
+keyedNodeWithText =
+    Keyed.ul [] [ ("0", Html.text "Hello!") ]
+
+
+keyedNodeWithTextDecoded : NodeType
+keyedNodeWithTextDecoded =
+    NodeEntry
+        { tag = "ul"
+        , children = [ TextTag { text = "Hello!" } ]
+        , descendantsCount = 1
+        , facts = emptyFacts
+        }
+
+
+keyedNodeWithChildren : Int -> Html.Html msg
```

Comment:
please move the helper functions either to another field or to the bottom of this one. The progression should be:

- base case triplets, where there's the string version, the Html version, and the decoded version
- the tests themselves, where it shows how each of these connect

- <rest>